spec/comment_spec.rb

```diff
@@ -45,6 +45,16 @@
     it "can see its child" do
       @comment.children.first.should == @child
     end
+    
+    it "can identify that an admin has replied to the parent" do
```

Comment:
```suggestion
    it 'can identify that an admin has replied to the parent' do
```
Could use single quotes in the 'it' descriptions.